candy-machine/program/src/processor/update.rs

```diff
@@ -61,6 +60,13 @@ pub fn handle_update_candy_machine(
         return err!(CandyError::CannotSwitchToHiddenSettings);
     }
 
+    if candy_machine.data.items_available > 0
```

Comment:
I think we can remove this condition `candy_machine.data.items_available > 0`; otherwise you could technically set the `items_available` to `0` and then switch from hidden settings. There is no scenario where you can switch from hidden settings, given that the account size does not have space for config lines.

Reply:
I agree

Reply:
True. Thanks for the input.
Done.